src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -190,6 +190,7 @@
         <thead>
             <tr>
                 <th>Locale</th>
+                <th>Fallback locale</th>
```

Comment:
it's only useful when rendering `fallback_messages` isnt it? perhaps add an argument to toggle the column as such

Reply:
yep, updated it.